test/e2e/specs/tracks/tracks__lohp-to-signup-events.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @group calypso-release
```

Comment:
I'm not sure which group to use here.
Since the test flows involve LOHP/landing pages as well as Calypso, I'm not sure if it makes sense to use `calyspo-release` or `calypso-pr`. I wonder if something like [these martech tests](https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e/specs/martech), that [send alerts to a Slack channel](https://github.com/Automattic/wp-calypso/blob/trunk/.teamcity/_self/projects/MarTech.kt#L110C10-L110C10), would make sense. 🤔 

Suggestions welcome.

Reply:
Good question. In general, if these tests require immediate feedback and/or test a critical path, we should run them for every commit (`calypso-pr`). If not, I think it's OK to run them against the merge commit only (`calypso-relese`).

FWIW, here's some more info on the test groups: https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/docs/tests_ci.md#featuretest-groups

Reply:
I don't think they need to be run for every commit.
Even running them against the merge commit I wonder about. It's possible that changes outside of Calypso could cause the tests to fail, like changing the logged-out homepage. In which case the next unlucky person to ship Calypso would see the failure even though it's unrelated to their change.

That's what makes me curious about the possibility of running tests at some preset cadence and sending failures to a Slack channel. Am I mistaken or is that how the `legal` group works (at least the martech tests using it)? Do you think it's reasonable to do something similar for these tests?

Reply:
> It's possible that changes outside of Calypso could cause the tests to fail, like changing the logged-out homepage. In which case the next unlucky person to ship Calypso would see the failure even though it's unrelated to their change.

There was an idea (p1635527872168500/1635432144.084600-slack-CQD1HH4MA) for creating a `daily` group but I don't think it was ever implemented. Having said that, I think it would also make sense to have a dedicated group for this suite, e.g. `lohp-tracking`. What do you think, @noahtallen @tyxla?

Reply:
LOHP is a pretty critical part of our flows, so I believe it might be a good idea to run the related tests periodically, daily, or similar. It could work with part of the tests that tests critical areas, it doesn't have to be all the tests.

Reply:
Thanks. If we create new daily or lohp/landing page specific group, is that something your team would typically handle? I'm happy to handle it or help where needed, but will likely need some pointers on where that's done.

Reply:
Historically, KitKat folks used to help with that, and we've supported them. That's no longer the case, and we're also stepping away from e2e work, leaving that to product folks, see:

- p58i-gh5-p2
- pLwCt-dDG-p2

We could of course help with reviews and consulting, but we no longer plan to prioritize working on new e2e tests or to actively maintain the e2e infrastructure.

Reply:
Gotcha. To add some mechanism that runs tests once per day, would [this](https://github.com/Automattic/wp-calypso/blob/trunk/.teamcity/_self/projects/MarTech.kt#L110C10-L110C10) TeamCity MarTech project be a good example to model it after?

Or possibly adding it to that project since this is sort of MarTech related?

I'm new to these configurations so any direction you could give would be very much appreciated.

Reply:
I think this is a good example - feel free to either use that project or duplicate it. Not sure how much it makes sense to use it since it's serving a very specific Martech purpose - capturing screenshots of where we render ToS. I'd recommend starting with a new one and altering it to your needs.

Reply:
Thanks @tyxla – some other priorities have come up and so adding a new group might have to wait (probably/hopefully not too long).

Do you or @WunderBart have any objections to deploying this using the `calyspo-release` group for now. I'll then circle back when I have time and add the new group.

Reply:
No objections from my side 👍